packages/angular_devkit/core/src/json/schema/registry.ts

```diff
@@ -457,10 +457,8 @@ export class CoreSchemaRegistry implements SchemaRegistry {
         valid: true,
         compile: (schema, _parentSchema, it) => {
           const compilationSchemInfo = this._currentCompilationSchemaInfo;
-          if (compilationSchemInfo === undefined) {
+          if (compilationSchemInfo === undefined || compilationSchemInfo === null) {
             throw new Error('Invalid JSON schema compilation state');
```

Comment:
`return () => true;` here instead of the exception and the null potential can be removed completely.